spec/bolt_spec/plan_spec.rb

```diff
@@ -35,6 +35,11 @@
   it 'fails when not stubbed' do
     expect { run_plan(plan_name, 'nodes' => targets) }.to raise_error(RuntimeError, /Unexpected call to/)
   end
+
+  it 'prints expected parameters when erroring' do
+    expect_action.not_be_called
+    expect { run_plan(plan_name, 'nodes' => targets) }.to raise_error(RuntimeError, /with parameters/)
```

Comment:
This should also include a test that the parameters are part of the error message. It's hard to do for the reused test cases, but I think putting a specific test to look for the correct parameters  in the `context 'with tasks'` section would be useful.